app/Http/Controllers/ControllerDash.php

```diff
@@ -102,8 +102,8 @@ public function dash() {
             $trainer_by_cid[$t['cid']] = $t['name'];
         }
         arsort($trainer_by_total);
-        foreach($trainer_by_total as $trainer_cid => $tt) {
-            if($tt == 0) {
+        foreach ($trainer_by_total as $trainer_cid => $tt) {
+            if ($tt == 0) {
```

Comment:
Another reformat error - please update and revert to previous... out of scope for this PR.